packages/mui-material/src/DialogTitle/DialogTitle.d.ts

```diff
@@ -28,4 +36,11 @@ export interface DialogTitleProps extends StandardProps<React.HTMLAttributes<HTM
  *
  * - [DialogTitle API](https://mui.com/material-ui/api/dialog-title/)
  */
-export default function DialogTitle(props: DialogTitleProps): JSX.Element;
+declare const DialogTitle: OverridableComponent<DialogTitleTypeMap>;
+
+export type DialogTitleProps<
+  D extends React.ElementType = DialogTitleTypeMap['defaultComponent'],
+  P = {},
```

Comment:
```suggestion
  P = { component?: React.ElementType },
```

I got this from Joy, otherwise `DialogTitleProps` does not have `component` as a key.

Reply:
Gottcha, thanks Jun!